test/cloudtest/test_storage.py

```diff
@@ -226,10 +235,11 @@ def get_num_workers(mz: MaterializeApplication) -> str:
     wait(
         condition="condition=Ready",
         resource="pod",
-        label="storage.environmentd.materialize.cloud/size=16",
+        label="cluster.environmentd.materialize.cloud/workers=4",
     )
 
-    assert get_num_workers(mz) == "'16'"
+    assert get_num_workers(mz, cluster_id) == "'4 4 4 4'"
+    assert get_num_scale(mz, cluster_id) == "'4 4 4 4'"
```

Comment:
Is workers and scale always expected to have the same result?

Reply:
Ok, can answer it myself based on later commit: They are not.

Reply:
This is because SIZE 4-4 is used in this particular test. SIZE 4 = scale 1 , workers = 4 ; SIZE 4-1 = scale 4 , workers = 1 and SIZE 4-4 = scale 4, workers 4.

Reply:
What is important here is that get_num* returns X values if scale = X . This means that X pods have been started in K8s, which is what is expected for scale = X .